README.md

```
@@ -45,7 +45,8 @@ pnpm dev
 
 - **アプリ**: http://localhost:3000
 - **API ドキュメント**: http://localhost:3000/api/swagger
-- **型定義生成**: `pnpm generate-types`
+- **OpenAPI仕様**: http://localhost:3000/api/openapi.yaml
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)